### PR TITLE
Add glass, sss, and light osl shaders for 3ds max

### PR DESCRIPTION
**Pull request by usakhelo**
This pr is complementary to [Max PR](https://github.com/appleseedhq/appleseed-max/pull/100)

### CODE REVIEW

**Comment by usakhelo** [comment]
Totally not sure about SSS shader, may look super-stupid what I did there.
When Disney shader is a base material in blend material and Light shader is a layer material, it makes disney look dark. Not black but weirdly dark. Even with mix amount 0.

File: src/appleseed.shaders/src/max/as_max_light_material.osl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+shader as_max_light_material
+(
+    closure color        BSDF1 = 0,
```
Comment (dictoon) [inline src/appleseed.shaders/src/max/as_max_light_material.osl]: Why the 1 in BSDF1?

File: src/appleseed.shaders/src/max/as_max_sss_material.osl [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#include "appleseed/material/as_material_helpers.h"
+
+shader as_max_sss_material
+(
+// BSSRDF part.
```
Comment (dictoon) [inline src/appleseed.shaders/src/max/as_max_sss_material.osl]: Indent comments like arguments?

File: src/appleseed.shaders/src/max/as_max_sss_material.osl [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    float                SSSReflectance = 0.8,
+    color                SSSColor = 1.0,
+    string               Profile = "normalized_diffusion",
+    color                Radius = 1.0,
+    float                RadiusScale = 0.1,
+    float                Ior = 1.3,
+    int                  MaxSubsurfaceRayDepth = 2,
```
Comment (dictoon) [inline src/appleseed.shaders/src/max/as_max_sss_material.osl]: 1 is probably a better default.

File: src/appleseed.shaders/src/max/as_max_light_material.osl (hunk 1 above)
Comment (usakhelo) [reply src/appleseed.shaders/src/max/as_max_light_material.osl]: Leftover from the original shader. Probably should be "Zero" or something like that.

File: src/appleseed.shaders/src/max/as_max_sss_material.osl (hunk 3 above)
Comment (usakhelo) [reply src/appleseed.shaders/src/max/as_max_sss_material.osl]: Ok. This parameter is not connected to any ui param.

**Comment by dictoon** [comment]
Thanks!

**Pull request merged**